src/ITILSolution.php

```diff
@@ -327,7 +327,7 @@ public function post_addItem()
         if (
             $this->input["itemtype"] == 'Ticket'
             && $_SESSION['glpiset_solution_tech']
-            && ($this->input['_disable_auto_assign'] ?? false) === true
+            && ($this->input['_disable_auto_assign'] ?? true) === true
```

Comment:
Using a `true` value by default, will remove this existing behaviour for all calls.

You have to pass this flag explicitely in `ITILSolution` input when you want to disable this behaviour.
```suggestion
            && ($this->input['_disable_auto_assign'] ?? false) === true
```

Reply:
My bad, double negation kills me.

```suggestion
            && ($this->input['_disable_auto_assign'] ?? false) === false
```